assets/src/components/Dashboard.tsx

```diff
@@ -51,14 +51,28 @@ const Dashboard = (props: RouteComponentProps) => {
 
   const logout = () => auth.logout().then(() => props.history.push('/login'));
 
+  const [notificationMessage, setNotificationMessage] = useState('Papercups');
+
+  const INTERVAL = 2000; //Set flashing interval
+
+  const _toggleNotificationMessage = () => {
+    if (notificationMessage.startsWith('New message')) {
+      setNotificationMessage('Papercups');
+    } else {
+      setNotificationMessage(`New message${totalNumUnread === 1 ? '' : 's'}`);
+    }
+  };
+
+  useEffect(() => {
+    const interval = setTimeout(_toggleNotificationMessage, INTERVAL);
+  });
```

Comment:
I'm not too familiar with `useEffect` -- how does this work if there is no second argument here? will every re-render/update trigger this?

also, should we add a `clearTimeout` when this unmounts?

Reply:
##  useEffect()
`useEffect `will run whenever the component will update/mount.

## clearTimeout()
`clearTimeout` is not necessary as timeout removes itself after INTERVAL is finished so no need to call this function when the component unmounts. [this](https://stackoverflow.com/questions/7391567/when-using-settimeout-do-you-have-to-cleartimeout) for reference. 

## Performance Impact.
It will not impact performance much as only the title tag will be re-rendered. [this](https://stackoverflow.com/questions/24718709/reactjs-does-render-get-called-any-time-setstate-is-called#:~:text=No%2C%20React%20doesn%27t%20render,render%20as%20little%20as%20possible.) for reference.

## Will this be triggered again and again?
yes it will be when the timeout expires and the title changes.

I have tried it with `setInterval` method which resulted in inconsistencies after some time (irregularity in flash interval).

Additional optimization can be that the interval only be created if there are 1 or more unread messages. (will do in next commit)

Reply:
makes sense, thanks for the notes!